tasks/libs/common/constants.py

```diff
@@ -9,6 +9,12 @@
 ALLOWED_REPO_ALL_BRANCHES = ALLOWED_REPO_NON_NIGHTLY_BRANCHES.union(ALLOWED_REPO_NIGHTLY_BRANCHES)
 AGENT_VERSION_CACHE_NAME = "agent-version.cache"
 
+# Metric Origin Constants:
+# https://github.com/DataDog/dd-source/blob/a060ce7a403c2215c44ebfbcc588e42cd9985aeb/domains/metrics/shared/libs/proto/origin/origin.proto#L144
+ORIGIN_PRODUCT = 17
+ORIGIN_CATEGORY = 0
```

Comment:
I don't see `category` on the file you gave, do you have the corresponding enum somewhere?

Reply:
@spencergilbert and @FlorentClarret mentioned that we should use constants that aren't super clear so we don't leak anything (https://github.com/DataDog/datadog-agent/pull/30834#discussion_r1832717812).

`ORIGIN_PRODUCT` points to [this part of the file](https://github.com/DataDog/dd-source/blob/a060ce7a403c2215c44ebfbcc588e42cd9985aeb/domains/metrics/shared/libs/proto/origin/origin.proto#L146)

`ORIGIN_CATEGORY` points to [this part of the file](https://github.com/DataDog/dd-source/blob/a060ce7a403c2215c44ebfbcc588e42cd9985aeb/domains/metrics/shared/libs/proto/origin/origin.proto#L224)

`ORIGIN_SERVICE` points to [this part of the file](https://github.com/DataDog/dd-source/blob/a060ce7a403c2215c44ebfbcc588e42cd9985aeb/domains/metrics/shared/libs/proto/origin/origin.proto#L323)

the `category` is actually the subproduct

Reply:
As a consequence would it be `ORIGIN_CATEGORY = 29`? Not sure 😄

Reply:
Asked the origins team and they confirmed it should be 29